### DISCUSSION TITLE
Single request presentation of response time percentiels

### DISCUSSION THREAD

**Discussion by MoritzSalein** [ludeknovy/jtl-reporter#123] [Q&A] [answered]
The current implementation of a single request with its response time percentiels are stacked area charts. The first percentiels is the base area, each additional percentiels shows an area in addion to the base. This presentation seems wrong. In my opinion the P90 must be a area from the bottom. The P95 usually a litte bit higher and behind the P90 and P99 a little bit higher and behind the both others.  In the first screenshot you'll see the current implementation. And in the second screenshot with the picture manipulation my expectation. The differences I have explained are best seen in the labelling of the graphs.
![1 Screenshot](https://user-images.githubusercontent.com/83296020/151958103-7ca86133-dd9a-4237-a605-7ed4151d3b12.png)
![2 Screenshot](https://user-images.githubusercontent.com/83296020/151958378-2673e2be-fd7a-4da7-b011-bd06a37ed030.png)

**Comment by ludeknovy** [comment, marked as answer]
Good catch, creating an issue.

**Reply by ludeknovy** [reply]
How about this without stacking @MoritzSalein ?

![Screenshot 2022-02-01 at 15 20 52](https://user-images.githubusercontent.com/13610612/151985497-24acd07e-07ae-46d6-a6dc-7db4616bb665.png)

**Reply by MoritzSalein** [reply]
Yes, that would be nice :-)
